### ISSUE TITLE
Gene review (phenotypes)

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/clin.iobio#415]
When reviewing variants, it is important to be able to evaluate the role of the variant, but also how well the gene matches to the entered phenotypes. I imagine 2 things to do:

1. In the variant review panel in gene.iobio, there is info on gene associations. From here there should be a button to click to get more info. When clicked, it requests information from clin on the patient phenotypes. This will launch a modal to summarize information. The same modal can be launched from genepanel, but it. has the information readily available.
2. Design the modal. This should consider HPO, GTR, Phenolyzer info, the patient phenotypes and help summarize how relevant the gene is to the patient. This can be a larger project, incorporating info from patient matching project, bringing Barry and others in to help refine the design, but we should be able to come up with a starting point.

**Comment by AlistairNWard** [comment]
@adityaekawade Here is a first thought - which I'll put in the grant on gene.iobio.
1. We pull HPO associations and diseases associations for the gene from HPO
2. In the variant review panel of gene, we have an HPO section and we show the info under Patient Terms in the image below in this section. Here, we show the entered patient HPO terms. If the HPO term is also associated with this gene, we have the match icon. We also look to see if the entered HPO term is closely related to any of the HPO terms associated with the gene. If so, we show the relationship (e.g. child) between the patient HPO and the gene HPO terms
3. You can click to open the modal below to see all the info

![Asset 14](https://user-images.githubusercontent.com/523111/131893069-c2f0f273-0879-4b81-8d1f-222d62485d44.png)

**Comment by adityaekawade** [comment]
Looks good! 
1. Resource for HPO gene association: https://ci.monarchinitiative.org/view/hpo/job/hpo.annotations/lastSuccessfulBuild/artifact/rare-diseases/util/annotation/genes_to_phenotype.txt 
Is there an API or downloadable data to pull disease association for genes?
2. It should be easy to match the entered HPO phenotypes using the HPO ids. And we can keep the relations limited to parent- child hierarchy.

**Comment by AlistairNWard** [comment]
Yes, let's limit to parent-child. We can always expand at a later time

**Comment by AlistairNWard** [comment]
I made a minor update to the image.
1. Include how many patient terms are entered in the details
2. Made it clear that all patient HPO terms are shown, even if there is no match. It is important that users know how many terms were unmatched as well as matched
3. Also included what we could show in the gene.iobio variant summary view

![Asset 15](https://user-images.githubusercontent.com/523111/132128034-859181e3-5945-414a-b253-58188c95153b.png)
